cmd/broker.go

```diff
@@ -0,0 +1,211 @@
+	SpecCount  int `json:"spec_count"`
+	ImageCount int `json:"image_count"`
+}
+
+var brokerName string
+
+var brokerCmd = &cobra.Command{
+	Use:   "broker",
+	Short: "Interact with an Automation Broker instance",
+	Long:  `List or Bootstrap bundles on an Automation Broker instance`,
+}
+
+var brokerCatalogCmd = &cobra.Command{
+	Use:   "catalog",
+	Short: "List available service bundles in broker catalog",
+	Long:  `Fetch list of service bundles in Automation Broker catalog`,
+	Run: func(cmd *cobra.Command, args []string) {
+		listBrokerCatalog()
+	},
+}
+
+var brokerBootstrapCmd = &cobra.Command{
+	Use:   "bootstrap",
+	Short: "Bootstrap an Automation Broker instance",
+	Long:  `Refresh list of bootstrapped service bundles in Automation Broker catalog`,
+	Run: func(cmd *cobra.Command, args []string) {
+		bootstrapBroker()
+	},
+}
+
+func init() {
+	brokerCmd.PersistentFlags().StringVarP(&brokerName, "name", "n", "openshift-automation-service-broker", "Name of Automation Broker instance")
+	rootCmd.AddCommand(brokerCmd)
+
+	brokerCmd.AddCommand(brokerCatalogCmd)
+	brokerCmd.AddCommand(brokerBootstrapCmd)
+}
+
+func listBrokerCatalog() {
+	log.Debugf("func::listBrokerCatalog()")
+	kube, err := clients.Kubernetes()
+	if err != nil {
+		log.Errorf("Failed to connect to cluster: %v", err)
+		return
+	}
+
+	brokerRoute, err := getBrokerRoute(brokerName)
+	if err != nil {
+		log.Errorf("Failed to get broker route: %v", err)
+	}
+
+	osbConf := &osb.ClientConfiguration{
+		Name:                "automation-broker",
+		URL:                 brokerRoute,
+		APIVersion:          osb.LatestAPIVersion(),
+		TimeoutSeconds:      60,
+		EnableAlphaFeatures: false,
+		Insecure:            true,
+		AuthConfig: &osb.AuthConfig{
+			BearerConfig: &osb.BearerConfig{
+				Token: kube.ClientConfig.BearerToken,
+			},
+		},
+	}
+	osbClient, err := osb.NewClient(osbConf)
+	if err != nil {
+		log.Errorf("Failed to make osb client: %v", err)
+		return
+	}
+
+	services, err := osbClient.GetCatalog()
+	printServices(services.Services)
+	return
+}
+
+func bootstrapBroker() {
+	kube, err := clients.Kubernetes()
+	if err != nil {
+		log.Errorf("Failed to connect to cluster: %v", err)
+		return
+	}
+
+	// Get the broker route given brokerName
+	brokerRoute, err := getBrokerRoute(brokerName)
+	if err != nil {
+		log.Errorf("Failed to get broker route: %v", err)
+		return
+	}
+
+	// Create a new bootstrap request
+	req, err := http.NewRequest("POST", fmt.Sprintf("%v/v2/bootstrap", brokerRoute), nil)
+	if err != nil {
+		log.Errorf("Failed to create request: %v", err)
+		return
+	}
+
+	// Set Bearer token Auth
+	req.Header.Set("Authorization", fmt.Sprintf("Bearer %v", kube.ClientConfig.BearerToken))
+
+	// Set TLS settings
+	cfg := &tls.Config{
+		InsecureSkipVerify: true,
+	}
+	http.DefaultClient.Transport = &http.Transport{
+		TLSClientConfig: cfg,
+	}
+
+	// Do bootstrap request
+	fmt.Printf("Attempting to contact the broker at %v/v2/bootstrap...\n", brokerRoute)
```

Comment:
As a user, when I saw this message I thought that the attempt to contact the broker may have stalled out since it took 10+ seconds. The text output didn't cue me into the fact that the `bootstrap` was potentially running. On slower internet, I suspect the process can take longer. I don't think we have any means to give a progress indicator here (correct me if I'm wrong).

I'd suggest having an alternative message here, something like:
```
Bootstrapping the broker at [...]
or 
Bootstrapping the broker at [...]. This may take a few moments...
or 
Attempting to bootstrap the broker at [...]
```